components/scheduler/views/timeline.md

```diff
@@ -19,9 +19,22 @@ In this article:
 * [Example](#example)
 * [Resource Grouping](#resource-grouping-in-the-timeline-view)
 
-@[template](/_contentTemplates/scheduler/views.md#day-views-common-properties)
+## View Parameters
+
+Generally, the views are designed around the timeframe that they show and the day-based views share some common properties that you will likely have to set to provide a good user experience for the user:
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Attribute | Type and Default Value | Description |
+|----------|----------|----------|
 | `ColumnWidth` | `decimal` | The width of each time column in pixels.
+| `EndTime` | `DateTime` | The counterpart to `StartTime` - defines when the full day ends. Defaults to midnight. If you have the day end earlier you can reduce the amount of elements that render, but the user may not see some late appointments.
```

Comment:
* Use an em dash.
* "If you have the day end earlier" -> If you set the `EndTime` earlier than midnight, you reduce the elements that render, but the user may not see late appointments.